lib/src/controls/surfaces/expander.dart

```diff
@@ -196,87 +190,87 @@ class ExpanderState extends State<Expander>
       // HEADER
       HoverButton(
         onPressed: _handlePressed,
-        builder: (context, states) {
-          return Container(
-            height: widget.headerHeight,
-            decoration: ShapeDecoration(
-              color: widget.headerBackgroundColor?.resolve(states) ??
-                  theme.resources.cardBackgroundFillColorDefault,
-              shape: widget.headerShape?.call(open) ??
-                  RoundedRectangleBorder(
-                    side: BorderSide(
-                      color: theme.resources.cardStrokeColorDefault,
-                    ),
-                    borderRadius: BorderRadius.vertical(
-                      top: const Radius.circular(4.0),
-                      bottom: Radius.circular(open ? 0.0 : 4.0),
-                    ),
+        builder: (context, states) => Container(
```

Comment:
use `return` instead of `=>`

Reply:
okay.

Reply:
this has been implemented in the latest commit [54fde58]